javascript/src/URDFLoader.js

```diff
@@ -151,6 +151,7 @@ class URDFLoader {
         const manager = this.manager;
         const linkMap = {};
         const jointMap = {};
+        const jointList = [];
```

Comment:
Rather than bookkeep a separate jointList I think I'd rather use `Object.values( jointMap )` when we need to iterate over all joints.

Reply:
No problem, I figured this was a bit janky when I was writing it. I didn't know about `Object.values( x )`. Thanks, I'll use that in the future.